functions/auth0_cis_webhook_consumer/config.py

```diff
@@ -1,27 +1,42 @@
 import logging
 import os
+import json
 from typing import Optional
 
 import requests
 import boto3
+from botocore.exceptions import ClientError
 
 logger = logging.getLogger(__name__)
 
-
-def get_paginated_results(product, action, key, credentials=None, args=None):
-    args = {} if args is None else args
-    credentials = {} if credentials is None else credentials
-    return [y for sublist in [x[key] for x in boto3.client(
-        product, **credentials).get_paginator(action).paginate(**args)]
-            for y in sublist]
-
+def get_secret_value(self, path, secret_name):
+    #if the secret exists return it
+    print(path, secret_name, self._secrets)
```

Comment:
Is this a debug line, or...?
Feels like this is going to leak `_secrets` somewhere they shouldn't go.

Reply:
Forgot to remove that. Good catch.